README.md

```diff
@@ -8,9 +8,18 @@ server] support. It's a flexible frontend for parts of the [symbolic] library.
 
 [Documentation]
 
+## Contained Binaries
+
+This repository contains several binary crates that can be run with `cargo run -p <PACKAGE>`:
+
+* `symbolicator`: Runs a symbolicator server. Calling `cargo run` without a package will run this binary.
+* `symsorter`: Sorts debug files into a structure usable by `symbolicator`.
+* `wasm-split`: Adds build IDs to wasm files.
+* `symbolicli`: Performs symbolication on the CLI. See [the readme](./crates/symbolicli/README.md).
+
 ## Compiling
 
-Symbolicator is currently not distributed as binary, which means you need to
+Symbolicator is currently not distributed as a binary, which means you need to
```

Comment:
it kinda is though?

Reply:
Yes, we do build and distribute binaries: https://github.com/getsentry/symbolicator/releases/tag/0.6.0